src/utils/errors.ts

```diff
@@ -0,0 +1,63 @@
+import { EthersError, PriceFetchingError, SdkError, SimulationError, TenderlyError, ZapperError } from '@yfi/sdk';
```

Comment:
Lets create a `types/Error.ts` and re-export these error from there. We can later continue adding new types of errors from that single source.

Reply:
btw this are not types, but the actual classes